api/DOMRect.json

```diff
@@ -74,10 +74,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "31"
```

Comment:
This matches https://developer.mozilla.org/en-US/docs/Web/API/DOMPoint#Browser_compatibility so is almost certainly correct. (Or both are wrong, but you've already confirmed manually.)